server/neptune/temporalworker/job/stream_handler.go

```diff
@@ -15,43 +14,51 @@ type OutputLine struct {
 	Line  string
 }
 
+type StreamHandler interface {
+	Stream(jobID string, msg string)
+	Handle()
+	CloseJob(ctx context.Context, jobID string) error
+	CleanUp(ctx context.Context) error
+}
+
+// TODO: Return streaming channel
```

Comment:
is there anything that blocks doing this now?

Reply:
Ah, forgot to remove this comment from debugging. 

I was exploring creating the jobs output channel here and passing it to back to the caller but since we close the main channel in server, we should be creating it in the server as well. So, I chose to pass in the jobs output channel to the constructor.